target/linux/ath79/image/generic.mk

```diff
@@ -196,10 +196,10 @@ endef
 
 
 define Device/8dev_carambola2
-  SOC := ar9331
+  	SOC := ar9331
```

Comment:
Please remove all changes to generic.mk, that are not related to the device you want to add support for.